apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```diff
@@ -67,94 +52,6 @@ import { IUser, TimesheetLog, TimesheetStatus } from '@/app/interfaces';
 import { toast } from '@components/ui/use-toast';
 import { ToastAction } from '@components/ui/toast';
 
-export const columns: ColumnDef<TimeSheet>[] = [
```

Comment:
@Innocent-Akim why that whole block was added and now removed? What was function of it? Did you replaced it with something else or it was not needed at all or what?

Reply:
@evereq  Here, I simply removed the table headers. With the new design, I realized that using a Shadcn table was no longer necessary.